javatests/google/registry/flows/domain/DomainCreateFlowTest.java

```diff
@@ -437,6 +437,18 @@ public void testSuccess_generalAvailability_ignoresEncodedSignedMark() throws Ex
   }
 
   @Test
+  public void testSuccess_superuser_ignoresEncodedSignedMark() throws Exception {
```

Comment:
Can you elaborate the test name to "ignoresEncodedSignedMarkMismatch" to be a bit more specific about what's being ignored?